lib/actions/help_ts.ts

```diff
@@ -15,32 +15,37 @@
  * limitations under the License.
  */
 
-import { Command } from '@oclif/command';
-import * as Bluebird from 'bluebird';
 import * as _ from 'lodash';
 import * as path from 'path';
+import Command from '../command';
 
 import { capitanoizeOclifUsage } from '../utils/oclif-utils';
 
-export async function getOclifHelpLinePairs(): Promise<
-	Array<[string, string]>
-> {
+export async function getOclifHelpLinePairs(
+	primary: boolean,
```

Comment:
@pdcastro Right, I was aware of this, but had tested with `time` and could not see any difference (and also couldn't perceive and delay before help was shown, since time is less useful because of the delay at the end).  So I left it because I didn't think it was worth further refactoring for no benefit.  Can take another look though, if you want.

Reply:
I've refactored this now.